src/main/java/adris/altoclef/AltoClefCommands.java

```diff
@@ -31,12 +63,42 @@
 import net.minecraft.util.registry.Registry;
 import net.minecraft.world.chunk.EmptyChunk;
 
-import java.io.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.List;
 
+
 /// This structure was copied from a C# project. Fuck java. All my homies hate java.
 public class AltoClefCommands extends CommandList {
 
+    public AltoClefCommands(CommandExecutor executor) throws CommandException {
+        super(executor,
+              // List commands here
+              new HelpCommand(),
+              new GetCommand(),
+              new FollowCommand(),
+              new GiveCommand(),
+              new EquipCommand(),
+              new GotoCommand(),
+              new CoordsCommand(),
+              new StatusCommand(),
+              new InventoryCommand(),
+              new LocateStructureCommand(),
+              new StopCommand(),
+              new TestCommand(),
+              new FoodCommand(),
+              new ReloadSettingsCommand(),
+              new GamerCommand(),
+              new PunkCommand()
+              //new TestMoveInventoryCommand(),
+              //    new TestSwapInventoryCommand()
+             );
+    }
+
+    // TODO: 2021-05-22 move to unit tests
```

Comment:
Unit tests might be nice, this is here more so as a "playground" for me to quickly test stuff and then delete it.

Should probably put playground stuff in a separate file.